extensions/wikia/CreatePage/js/CreatePage.js

```diff
@@ -222,6 +222,8 @@ var CreatePage = {
 						label: 'modal'
 					} );
 
+					window.dispatchEvent( new Event('CreatePageModalOpened') );
```

Comment:
Tracking is attached to the impression of modal redlinks in [Special:AbTesting](http://leagueoflegends.warkot.wikia-dev.com/wiki/Special:AbTesting). Code in the AB Testing listens for this event. When the event fires, code checks if there are redlinks returned from the backend and if the user is not in the control group (control group has redlinks hidden).